Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-2064.dfy.expect

```diff
@@ -0,0 +1,35 @@
+git-issue-2064.dfy(13,9): Error: to be a refinement of abstract type 'UnitTestParent.T', type synonym 'OpaqueType.T' must support the type characteristic "no references"
```

Comment:
```suggestion
git-issue-2064.dfy(13,9): Error: to be a refinement of abstract type 'UnitTestParent.T' declared `(!new)`, type synonym 'OpaqueType.T' must contain no references
```

Reply:
Great suggestions about the wording of the error messages. I have made the changes.